### ISSUE TITLE
Unable to perform chunk uploading in mobile browser (chrome)

### ISSUE THREAD

**Issue by mohsen-zanjani** [pqina/filepond#401]
# Summary
Hi @rikschennink ...

I have a project where the upload is done by chunk uploading and it works well on desktop browsers. But in mobile browsers, nothing happens after receiving the response of the POST request from the server (i.e unique location id). This means that patch requests are not sent at all and there are no errors.
However, when **chunkUploads** is set to **false**, everything works correctly.

Is the chunk uploading process in filepond supported by mobile browsers?
Have you tested Chunk uploading on mobile browsers?

Can you give me a hint about how it works and what can I do? ...
Thanks ...

This is the server settings:
```js
myServer: {
    url: '/api/chunk-upload/',
    headers: {
        "X-CSRF-Token": document.head.querySelector('meta[name="csrf-token"]').content,
    },
    process: {
        method: 'POST',
        withCredentials: false,
        onload: response => {
            console.log(response);
            if (response instanceof XMLHttpRequest)
                return response.responseText;

            return response;
        },
        onerror: response => {
            console.log(response);
        }
    },
    restore: null,
    load: null,
    fetch: null,
}
```

### How to reproduce
I've used the FilePond PHP Boilerplate for the back-end (converting the code to the Laravel structure) as well as the Vue adapter of filepond for the front-end.

### Expected behaviour
Perform chunk uploading on mobile browsers (specifically Chrome) as this process is done correctly on desktop browsers

### Additional information
I'm not sure if this is a bug or just my mistake. But the behavior that file uploading (chunk uploading) is doing good in the desktop browser, but not in the mobile browser, is a bit strange to me. On the other hand, regular uploads (without chunk uploading) are done correctly in both browsers.
Also, when I check the console using the [Eruda ](https://github.com/liriliri/eruda) tool, only one **null** word is printed in red (as error) and no other error is displayed.

| Environment  | Version
|------------- | -----------
| OS           | Android 9
| Device       | Samsung Galaxy A50
| Browser      | Chrome 77

**Comment by rikschennink** [comment]
I've tested on iOS and it works correctly there, now running tests on mobile Chrome.

**Comment by rikschennink** [comment]
Tested successfully with Samsung Galaxy s10 on BrowserStack.

**Comment by mohsen-zanjani** [comment]
First, thank you for your quick reply.
Yes, you are right. chunk uploading is working properly and the problem is elsewhere which is strange to me and I didn't even think about it.

I intended to use filepond to upload video files (MP4). The tests on my laptop were working properly. For testing on mobile, I only used video files that I recorded with my phone camera, which unfortunately did not complete the chunk uploading process and I had no errors!

After your reply, I realized the problem was probably from my code. But after a lot of code review, I didn't get any results. Eventually, I decided to copy one of the phone's videos to my laptop and run the tests there.  I hoped there would be a difference in displaying the errors.
and, yes it was, and I finally saw an error in the console ... which you can see below.

![filepond](https://user-images.githubusercontent.com/31995071/67869433-a9ff3980-fb42-11e9-99a2-61d8b3b24ee9.PNG)


**Actually the problem was in the file name !!!**
The files that I tested on my phone were saved in my native language (Persian, something like this: **تست.mp4** ) because of the default phone settings. After checking a few files on my laptop and mobile I was sure that the problem was exactly in the file name.

Also, given the error displayed on the console, I think the error is related to this part of the filepond code.

   ```js
 // add headers
    Object.keys(options.headers).forEach(function(key) {
      xhr.setRequestHeader(key, options.headers[key]);
    });
```

I don't know why this error should occur. On the other hand, I don't know if this is reasonable behavior in filepond or not. I didn't see anything in the Documents and Issues section either. 
**Note that this problem only occurs when chunk uploading is enabled.**

Do you have any idea about this? Did I make a mistake again and had to set something up (eg Unicode) and forget it? Note that I also did this test on the FilePond PHP Boilerplate, without any changes to the code, and the above error occurred again (I just added this to the code `chunkUploads: true,`).

Thanks again for your response ...

**Comment by rikschennink** [comment]
Should be fixed in version `4.7.4`, can you confirm?

**Comment by mohsen-zanjani** [comment]
Yes ... I tested it ... everything works fine ... Thank you very much ...
